superset-frontend/src/views/CRUD/data/hooks.ts

```diff
@@ -80,6 +81,7 @@ export const UseGetDatasetsList = (queryParams: string | undefined) =>
     endpoint: `/api/v1/dataset/?q=${queryParams}`,
   })
     .then(({ json }) => json)
-    .catch(error =>
-      logging.error('There was an error fetching dataset', error),
-    );
+    .catch(error => {
+      addDangerToast('There was an error fetching dataset');
```

Comment:
```suggestion
      addDangerToast(t('There was an error fetching dataset'));
```

Reply:
Fixed in [`this commit`](https://github.com/apache/superset/pull/22610/commits/a72d05829371f6a375e165dd2d3739474587fa63).